### PR TITLE
Winchestery

### PR DESCRIPTION
**Pull request by Winchestery**
MVC components:(Partners, Contacts) added
styles.scss changed
footer fixed

### CODE REVIEW

File: app/components/partners/partners-model.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default class PartnersModel {
```
Comment (PavelKundenko) [inline app/components/partners/partners-model.js]: You don't use this data, so we don't need model here, you can remove it :)

File: app/components/contacts/contacts-model.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default class ContactsModel {
```
Comment (PavelKundenko) [inline app/components/contacts/contacts-model.js]: Remove empty model, please

File: app/components/contacts/contacts-controller.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  constructor({ subscribe, events }) {
+    this.view = new ContactsView();
+    this.model = new ContactsModel();
+
+    this.subscribe = subscribe;
+    this.events = events;
+
+    this.subscribe(this.events.SHOW_CONTACTS, this.view.render);
+    
+    // this.onLoad();
```
Comment (PavelKundenko) [inline app/components/contacts/contacts-controller.js]: Remove redundant comments, please

File: app/components/partners/partners-controller.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/extensions
+import PartnersModel from './partners-model.js';
+// eslint-disable-next-line import/extensions
```
Comment (PavelKundenko) [inline app/components/partners/partners-controller.js]: Dev branch has a fix for this rule, so it's better to remove these comments and pull changes from dev

File: app/components/contacts/contacts-styles.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
+@import "../../css/main.scss";
```
Comment (PavelKundenko) [inline app/components/contacts/contacts-styles.scss:1]: Use single quotes, please

**Review by MarinaPasternak — APPROVED** [review]

File: dist/index.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@1,300;1,400&family=PT+Serif:ital,wght@0,400;1,700&display=swap" rel="stylesheet">
+  <title>Car shop</title>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Car shop</title>
```
Comment (PavelKundenko) [inline dist/index.html]: Meta tags and title repeats two times

File: app/components/partners/partners-controller.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  constructor({ subscribe, events }) {
+    this.view = new PartnersView();
+    this.model = new PartnersModel();
+
+    this.subscribe = subscribe;
+    this.events = events;
+
+    this.subscribe(this.events.SHOW_PARTNERS, this.view.render);
+
+    this.onLoad();
```
Comment (PavelKundenko) [inline app/components/partners/partners-controller.js]: This component shouldn't be rendered with page loading. It should render only when receives SHOW_PARTNERS event.

**Comment by Winchestery** [comment]
issues have been fixed

**Review by PavelKundenko — APPROVED** [review]

**Review by rearchontis — APPROVED** [review]

**Review by MozdolevskyiMaksym — APPROVED** [review]

**Review by iklopov — APPROVED** [review]

File: app/components/contacts/contacts-view.js [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+				    	<iframe class="mx-auto" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2648.795710056671!2d35.01088877698998!3d48.40286621578827!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40dbfb496db41bcf%3A0x73dc0b934d13bc0a!2z0YPQuy4g0J_QvtCx0LXQtNGLLCA0MCwg0JTQvdC40L_RgNC-LCDQlNC90LXQv9GA0L7Qv9C10YLRgNC-0LLRgdC60LDRjyDQvtCx0LvQsNGB0YLRjCwgNDkwMDA!5e0!3m2!1sru!2sua!4v1602761195902!5m2!1sru!2sua" width="1200" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe>
+				  </div>
+				  <div class="callMeBack">
+					    <img class="px-3 pb-3 pt-3" src="https://img.icons8.com/dotty/60/000000/phone.png"/>
+				   </div>
+			</section>`;
+
+		this.addEvent();
+	}
+		addEvent =()=> {
```
Comment (so2niko) [inline app/components/contacts/contacts-view.js]: add spaces `= () =>`

**Review by so2niko — COMMENTED** [review]
I want to see screenshots or video/animations of working components

File: app/components/partners/partners-view.js [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+  main = document.querySelector('.main');
+
+  render() {
+    this.main.innerHTML = `
+        <section>
+        	<h2 class="text-center pt-5">Our Partners</h2>
+	        <div class="mx-auto">
+                <div class="row d-flex align-items-center pt-2 mt-4">
+                    <div class="col-xs-10 col-sm-7 col-md-4 col-lg-2">
+                        <div>
```
Comment (so2niko) [inline app/components/partners/partners-view.js:13]: can optimize this code with arrays and loops?

File: app/main.js [hunk 10]
```diff
@@ -1,6 +1,11 @@
 import Publisher from './helpers/publisher.js';
-import { TemplateController } from './components/template/template-controller.js';
+import { ContactsController } from './components/contacts/contacts-controller.js'
+import { PartnersController }from './components/partners/partners-controller.js';
```
Comment (so2niko) [inline app/main.js:4]: add space `Controller }from`

**Review by so2niko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**